### DISCUSSION TITLE
Merge request rules

### DISCUSSION THREAD

**Discussion by atbakken** [GitbookIO/community#171] [Feature Requests]
### What's your feature/idea?

Would like to have the option to apply merge request rules to specific projects and/or specific pages. These rules could be named and would require specific number of approvals, or specific users to approve prior to allowing the merge.

### What's your desired outcome?

I would like to be able to setup multiple rules such as:

QA Rules - requires 2 number of approvers from the QA group/team
MGMT Rules - requires 2 number of approvers from MGMT group/team and require approval from specific user maybe a VP

Then I would like to apply QA rules to the QA project documents and at least 2 of the 6 QA members have approved before the change can be merged.

I would like to apply MGMT Rules to specific projects or specific pages and at least 2 of the 4 mgmt members have to approve, as well as a specific VP user, before the change can be merged.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by thebiglabasky** [comment]
Thanks for sharing! We're kicking off an initiative around better supporting review workflows, and this is definitely on the radar.
We'll keep this discussion updated with our progress to gather some feedback on how we're looking at things.
Stay tuned!
